pkg/event/raw.go

```diff
@@ -7,9 +7,9 @@ import (
 
 // Raw represents single event as received by an EventsProcessor instance
 type Raw struct {
-	Metadata          stringmap.StringMap
-	SloClassification *SloClassification
-	Quantity          float64
+	Metadata          stringmap.StringMap `json:"metadata"`
+	SloClassification *SloClassification  `json:"slo_classification"`
+	Quantity          float64             `json:"quantity"`
```

Comment:
Strictly speaking, we do not need them for most of the fields, except for `SloClassification`. However, I would prefer to keep things explicit.

Reply:
Oh, I missed that the `SloClassification` is used for the unmarshalling still. 
Personally I do not think we should use that for a definition of the Kafka JSON schema either and has custom struct for it in the Kafka ingester module package itself, so the schema is completely abstracted from the internal representation. 

And I'd avoid adding anything actually unused to the codebase since it would bring just possible confusion.

Reply:
https://github.com/seznam/slo-exporter/pull/48/commits/fc5b238d1592b75d8df45178c6943bdfeed35548